site/data/github-labels.yaml

```diff
@@ -346,4 +346,36 @@ default:
       target: issues
       addedBy: anyone
 
+# Labels about what category a PR is in terms of the release notes
+    - color: "164175"
+      description: A major change that needs more than a paragraph of explanation in the release notes.
+      name: release-note/major
+      target: prs
+      addedBy: prow
+    - color: "164175"
+      description: A minor change that needs about a paragraph of explanation in the release notes.
+      name: release-note/minor
+      target: prs
+      addedBy: prow
+    - color: "164175"
+      description: A small change that needs one line of explanation in the release notes.
+      name: release-note/small
+      target: prs
+      addedBy: prow
+    - color: "164175"
+      description: A documentation change for the release notes.
+      name: release-note/docs
+      target: prs
+      addedBy: prow
+    - color: "164175"
+      description: An infrastructure update for the release notes.
+      name: release-note/docs
```

Comment:
This might be a copy/paste error. Should be `release-note/infrastructure` or something similar?

Reply:
Yeah, thanks. Should be `release-note/infra`.